components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -142,8 +161,12 @@ export default function () {
 
             {showPersistentVolumeClaimUI && (
                 <>
-                    <br></br>
-                    <h3 className="mt-12">Workspaces</h3>
```

Comment:
**suggestion**: What do you think of moving the workspaces settings to the top of the general project settings here now that selecting a workspace class is a bit more prominent as it affects both workspaces and prebuilds?

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2022-11-09 at 6 16 05 PM" src="https://user-images.githubusercontent.com/120486/200887731-0f6777c0-fe9b-4e1f-8895-8a79b2304f91.png"> | <img width="1440" alt="Screenshot 2022-11-09 at 6 15 57 PM" src="https://user-images.githubusercontent.com/120486/200887735-8e93603f-ed60-4e8c-8145-3e8780495254.png"> |